### PR TITLE
Support an `offsetSeconds` accessor for Time and DateTime.

### PR DESCRIPTION
**Pull request by sherfert**

### CODE REVIEW

File: enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala [hunk 1]
```diff
@@ -697,7 +697,7 @@ class TemporalAcceptanceTest extends ExecutionEngineFunSuite with QueryStatistic
 
   test("should not provide undefined accessors for local time") {
     shouldNotHaveAccessor("localtime", Seq("year", "quarter", "month", "week", "weekYear", "day",  "ordinalDay", "weekDay", "dayOfQuarter",
-      "timezone", "offset", "offsetMinutes", "epochSeconds", "epochMillis",
+      "timezone", "offset", "offsetMinutes", "epochSeconds", "epochSeconds", "epochMillis",
```
Comment (Lojjs) [inline enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala]: You've just copied `epochSeconds` and forgot to change it into `offsetSeconds`

File: enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala [hunk 2]
```diff
@@ -708,7 +708,7 @@ class TemporalAcceptanceTest extends ExecutionEngineFunSuite with QueryStatistic
   }
 
   test("should not provide undefined accessors for local date time") {
-    shouldNotHaveAccessor("localdatetime", Seq("timezone", "offset", "offsetMinutes", "epochSeconds", "epochMillis",
+    shouldNotHaveAccessor("localdatetime", Seq("timezone", "offset", "offsetMinutes", "epochSeconds", "epochSeconds", "epochMillis",
```
Comment (Lojjs) [inline enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala]: Same as above

File: enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala [hunk 3]
```diff
@@ -719,7 +719,7 @@ class TemporalAcceptanceTest extends ExecutionEngineFunSuite with QueryStatistic
   test("should not provide undefined accessors for duration") {
     shouldNotHaveAccessor("duration", Seq("year", "quarter", "month", "week", "weekYear", "day",  "ordinalDay", "weekDay", "dayOfQuarter",
       "hour", "minute", "second", "millisecond", "microsecond", "nanosecond",
-      "timezone", "offset", "offsetMinutes", "epochSeconds", "epochMillis"), "{days: 14, hours:16, minutes: 12}")
+      "timezone", "offset", "offsetMinutes", "epochSeconds", "epochSeconds", "epochMillis"), "{days: 14, hours:16, minutes: 12}")
```
Comment (Lojjs) [inline enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalAcceptanceTest.scala]: Same as above

**Review by Lojjs — CHANGES_REQUESTED** [review]
Some copy paste errors in TemporalAcceptanceTest. You should also add `offsetSeconds` to the date test in that file.

File: community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java [hunk 4]
```diff
@@ -613,6 +617,15 @@ void assign( Builder<?> builder, AnyValue value )
                 throw new UnsupportedTemporalUnitException( "Not supported: " + name() );
             }
         },
+        offsetSeconds//<pre>
```
Comment (lutovich) [inline community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java:620]: What are these tags for?

Comment (sherfert) [reply community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java:620]: To be honest I just followed the pattern blindly. But @thobe should know maybe?

**Review by Lojjs — APPROVED** [review]

File: community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java (hunk 4 above)
Comment (thobe) [reply community/values/src/main/java/org/neo4j/values/storable/TemporalValue.java:620]: The tags are for telling intellij to not autoformat that code.

**Pull request merged**